backend/src/test/java/de/konstantintyker/audiophile/backend/artist/ArtistsTest.java

```diff
@@ -0,0 +1,31 @@
+package de.konstantintyker.audiophile.backend.artist;
+
+import de.konstantintyker.audiophile.backend.artists.Artist;
+import de.konstantintyker.audiophile.backend.artists.ArtistRepo;
+import de.konstantintyker.audiophile.backend.artists.ArtistService;
+import de.konstantintyker.audiophile.backend.artists.ArtistUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class TravelerServiceTest {
```

Comment:
please change the classname according to the filename of the unites:  from TravelerServiceTest to ArtistTest